modules/update-process-workflow.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-process-workflow_{context}"]
+= Update process workflow
+
+The following represents a detailed workflow of the {product-title} update process.
+
+. The intent to update the cluster is expressed through the web console or the CLI.
+The target version is stored in the `spec.desiredUpdate.version` field of the ClusterVersion resource.
+
+. The Cluster Version Operator (CVO) discovers that the `desiredUpdate` in the ClusterVersion resource differs from the current cluster version.
+Using the graph data from the OpenShift Update Service, the CVO resolves the desired cluster version to a pullspec to the release image.
+
+. The CVO validates the integrity and authenticity of the release image.
+Red Hat publishes cryptographically-signed statements about published release images at predefined locations, using image SHA digests as unique and immutable release image identifiers.
+The CVO utilizes a list of built-in public keys to validate the presence and signatures of the statement matching the checked release image.
+
+. The CVO creates a job named `version-$version-$hash` in the `openshift-cluster-version` namespace.
+This job uses containers that are executing the release image, so the cluster first downloads the image through the container runtime.
+The job then extracts the manifests and metadata from the release image to a shared volume that is accessible to the CVO.
+
+. The CVO validates the extracted manifests and metadata.
+
+. The CVO ensures that the cluster is in an upgradeable state.
```

Comment:
may be a bit confusing, since there's a cvo condition upgradeable=false, which (with some rare exceptions) only blocks minor (4.N+1) but not micro (4.13.N+1) updates

Reply:
Would it be worth adding a note below the following paragraph explaining that these conditions usually/almost always prevent minor updates and not patch updates?

Reply:
hmm.. it depends on the conditions..  
its just so happens to be, that we have cvo condition named "upgradeable", which is blocking (mostly) minors.  
however we may as well have Failing or Invalid conditions which may block either

Reply:
yeah basically my point is to avoid "_is in an **upgradeable** state_" wording, since it just happen to be cvo condition with the same name